### DISCUSSION TITLE
How to pass down additional props to Form

### DISCUSSION THREAD

**Discussion by KentonParton** [rjsf-team/react-jsonschema-form#2880] [Q&A] [answered]
Hi All,

I am using react-jsonschema-form with @rjsf/antd (Ant Design).

Off of Ant Designs Form component, they have a property named `size` which as the name suggests, controls the size of the different input fields (`Form.Item`). 

https://ant.design/components/form/#Form

I am wanting to include the `size` property off of react-jsonschema-form's `Form` property. I was hoping that react-jsonschema-form's `Form` would pass down any additional props to the Ant Design `Form` but that doesn't seems to be the case.

How would I go about including `size`?

Appreciate the help!

https://codesandbox.io/s/brave-poitras-jdshu0?file=/src/App.js

**Comment by heath-freenome** [comment, marked as answer]
From the looks of it, the `antd` theme isn't actually using the specific `Form` from Ant Design. Perhaps that is the issue. It is possible for you to customize your instance and use the `_internalFormWrapper` prop on the `@rjsf` `Form` to switch in the Ant Design form yourself... You would do something like the following (note this code is being typed in this editor and will need to be updated accordingly):

```
import Form from `@rjsf/antd`;
import { Form as AntdForm } from `antd`;
function MyForm(props) {
  return <AntdForm {...props} size={xxx} />;
}

<Form _internalFormWrapper={MyForm} schema={your_schema} />

```
